app/jobs/update_job.rb

```diff
@@ -13,7 +13,8 @@ class UpdateJob < ApplicationJob
   # @param [BackgroundJobResult] background_job_result
   # rubocop:disable Metrics/AbcSize
   # rubocop:disable Metrics/MethodLength
-  def perform(model_params:, signed_ids:, background_job_result:)
+  # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
this is a pretty long method with a bunch of branching and multiple ways to return.  but i didn't see any obvious ways to decompose it into something clearer.  the [`if start_workflow ...`](https://github.com/sul-dlss/sdr-api/pull/285/files#diff-2232f4d49d67b6a8ccaec48c4181a71e6e3cb0e28ec7d648aad63c22691dc35bR54-R58) conditional below was the last thing i added, and was what necessitated this exception.